### ISSUE TITLE
Can't view uninstalled Tools extension pages via keyboard

### ISSUE THREAD

**Issue by hawkticehurst** [microsoft/vscode#323507]
Testing #323371

Currently you can successfully navigate via keyboard to the Customizations modal Tools page and then get to the Tools marketplace (i.e. "Browse Marketplace" button). But once you're on the marketplace page you are only able to select/trigger the install button via keyboard. 

You cannot select the extension itself so you can view the extension information page via keybinding –– only when clicked.

**Comment by dmitrivMS** [comment]
Both `Enter` and `Space` work when the row is selected:

![Image](https://github.com/user-attachments/assets/e08f3bff-47af-4440-9b64-67b2be13693b)
